### ISSUE TITLE
Images are posted in the end

### ISSUE THREAD

**Issue by eqlion** [python273/telegraph#8]
Hello

Every time I want to post something using this wrapper images are posted in the end of an article even though I put them somewhere else.

For instance, executing this code:
```python
from telegraph import Telegraph

telegraph = Telegraph()

telegraph.create_account(short_name='test')

html = '''
<p>some text</p>
<img src='https://www.w3schools.com/css/img_fjords.jpg'/>
<p>more text</p>
'''

response = telegraph.create_page(
    'test',
    html_content=html)

print('http://telegra.ph/{}'.format(response['path']))
```
produces [this](http://telegra.ph/test-08-26-9) article. It doesn't matter whether I add `<figure><figcaption></figcaption></figure>` or not, all images are always after the text in random order.

**Comment by python273** [comment]
Fixed in 1.2

**Comment by eqlion** [comment]
thank you!

**Comment by python273** [comment]
One more fix in 1.2.1
